userguide/tutorials/aws-singletier.adoc

```diff
@@ -109,11 +103,11 @@ You will now be able to explore your instance and perform various configuration
 
 == Testing
 
-You can now try to login to KAUI from your browser using the URL http://INSTANCE_IP:443, where INSTANCE_IP is the IPV4 address for your instance, given on your dashboard as *Public IPV4 Address*. If all goes well, this should display the KAUI login screen. If this does *not* work, try https://INSTANCE_IP:443 (actually the :443 can be omitted). The browser may complain that your connection is not secure, but it should offer you a way to bypass the problem.
+You can now try to login to KAUI from your browser using the URL https://INSTANCE_IP:443, where INSTANCE_IP is the IPV4 address for your instance, given on your dashboard as *Public IPV4 Address*. If all goes well, this should display the KAUI login screen. The browser may complain that your connection is not secure, but it should offer you a way to bypass the problem.
```

Comment:
"The browser may complain that your connection is not secure, but it should offer you a way to bypass the problem."

Is this still true even though they are now using "https"?

Reply:
This is correct, unless a fully valid certificate is already in place.

Reply:
👍